blueprints/ember-cli-moment-shim/index.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   afterInstall: function() {
     // we need to install moment-timezone via bower since npmignore
     // ignores `moment-timezone/builds/*`
-    return this.addBowerPackagesToProject([
+    return this.addPackagesToProject([
```

Comment:
Just a note: I smoke-tested this in an app, and the vendored `moment` library is still being imported `ember-cli-moment-shim/node_modules` rather than `consuming-app/node_modules`.